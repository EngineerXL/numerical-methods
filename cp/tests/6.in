8130651869817364130684823
7826550296424402540968290
