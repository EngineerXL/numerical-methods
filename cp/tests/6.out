63634955801842714016230151705550594334751227262670
